src/WebUI/Components/KubeSummary.tsx

```diff
@@ -65,7 +66,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                 }
                 {
                     !!this.state.showDeployment &&
-                    this._getReplicaSetPodListComponent()
+                    this._getReplicaSetPodDetails()

```

Comment:
in service component also we will have POD, at that time also we need to show this view. anyway we have to optimize our flow, we can do this later also